src/pybind/mgr/dashboard/frontend/src/app/core/navigation/navigation/navigation.component.html

```diff
@@ -79,7 +79,7 @@
               *ngIf="permissions.configOpt.read">
             <a i18n
                class="dropdown-item"
-               routerLink="/configuration">Configuration Doc.
+               routerLink="/configuration">Configurations
```

Comment:
"Configurations" sounds somewhat weird. How about "Configuration settings" or "Configuration options"? This would be in line with how they are called in the [config documentation](http://docs.ceph.com/docs/master/dev/config/).

Or if that's too long, how about "Cluster -> Configuration" or "Cluster -> Settings"?

Reply:
Agree, I don't have a strong opinion about the naming. I'm fine with "Configuration options" or, if it's too long, "Configuration" or how about "Config options"?

Reply:
Let's stick to "Configuration" then - short and sweet. Thanks!

Reply:
Okay :)

Reply:
Done!